includes/class-publish-tweet.php

```diff
@@ -258,7 +258,7 @@ public function get_upload_data_media_id( $post ) {
 
 		$media_id = $this->upload( $file );
 
-		return $media_id ?: null;
+		return $media_id ? $media_id : null;
```

Comment:
The new WPCS made me do this.

Reply:
Yeah, they updated the standard, see: https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/